assets/app/views/_pod-template.html

```diff
@@ -48,6 +48,13 @@
     </div>
 
     <div ng-if="container.ports.length > 0" class="pod-template-ports">
+      <div>
+        <span ng-show="container.jolokiaUrl" class="pull-right">
```

Comment:
Seems like a good place to use https://github.com/openshift/origin/tree/master/assets#inject-additional-content-into-the-page